pkg/util/rolloutblock/rolloutblock.go

```diff
@@ -0,0 +1,58 @@
+package rolloutblock
+
+import (
+	"github.com/golang/glog"
+	"strings"
+
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+	shippererrors "github.com/bookingcom/shipper/pkg/errors"
+)
+
+// Return a boolean to decide if the application should override one of the
+// existing rollout blocks, a string representing the existing rollout blocks
+// that are not overridden, and an error if the application is trying to override
+// a non-existing rolloutblock
+func ShouldOverrideRolloutBlock(overrideRB string, nsRBs []*shipper.RolloutBlock, gbRBs []*shipper.RolloutBlock) (bool, string, error) {
+	overrideRBs := strings.Split(overrideRB, ",")
+	if len(overrideRB) == 0 {
+		overrideRBs = []string{}
+	}
+	RBs := append(nsRBs, gbRBs...)
+
+	nonOverriddenRBs, err := Difference(RBs, overrideRBs)
+
+	return len(nonOverriddenRBs) == 0, strings.Join(nonOverriddenRBs, ", "), err
```

Comment:
not sure if the second return argument is meant to be parsed somewhere, but it's concatenated with an extra space, which will make comma-splits all around to be broken. Better keep it consistent.